### ISSUE TITLE
StringTemplateIndent conflicts with MultilineRawStringIndentation

### ISSUE THREAD

**Issue by mrclrchtr** [detekt/detekt#7198]
## Expected Behavior
MultilineRawStringIndentation and StringTemplateIndent should work together.

## Observed Behavior
MultilineRawStringIndentation and StringTemplateIndent seems to conflict.

With fixed MultilineRawStringIndentation
![image](https://github.com/detekt/detekt/assets/16151098/62bf0621-28d6-45ef-8fef-14cd1e0f7110)

With fixed StringTemplateIndent
![image](https://github.com/detekt/detekt/assets/16151098/026b33f5-6352-4fb8-8c35-3f323e4653d0)

## Steps to Reproduce
MultilineRawStringIndentation
```yaml
  MultilineRawStringIndentation:
    active: true
    indentSize: 2
    trimmingMethods:
      - 'trimIndent'
      - 'trimMargin'
```
StringTemplateIndent
```yaml
  StringTemplateIndent:
    active: true
    autoCorrect: true
    indentSize: 2
```

```kotlin
    logger.info {
      """
        Postgres database login parameters:
        URL: ${container.jdbcUrl} 
        Username: ${container.username} 
        Password: ${container.password} 
      """.trimIndent()
    }
```

## Your Environment
* Version of detekt used: 2.4.0 (IntelliJ Plugin)

**Comment by schalkms** [comment]
One rule originates from ktlint wrapped in the formatting ruleset, one from the *native* style ruleset.
I don't know what detekt should handle here. What do you suggest?

Detekt includes multiple conflicting rules, where the user decides for one or the other rule to be enabled.

**Comment by schalkms** [comment]
Thank you for bringing this topic to our attention.
I added a clarifying comment to the detekt rule doc, which prevents further issues like this.

**Comment by mrclrchtr** [comment]
Thank you very much 😊
